ui/app/AppLayouts/Onboarding2/pages/KeycardEnterPinPage.qml

```diff
@@ -116,15 +133,13 @@ KeycardBasePage {
             }
             StateChangeScript {
                 script: {
-                    Backpressure.debounce(root, 100, function() {
-                        pinInput.clearPin()
-                    })()
+                    pinInput.statesInitialization()
```

Comment:
This should really do the whole `clearPin()`, and with a delay (on purpose)

Reply:
Not sure why I changed that :thinking: 
It might be a mistake on my part